main.js

```diff
@@ -0,0 +1,90 @@
+
+// Task #1
+
+// document.querySelector('#first-btn').onclick = transformToСentury;
+// const outFirstTask = document.querySelector('.first-task');
+
+// function transformToСentury() {
+//   const value = document.querySelector('.first-input').value; 
+//   const century = +value.slice(0, 2) + 1;
+//   const getEnding = function (d) {
+//     if (d > 3 && d < 21) return "th";
+//     switch (d % 10) {
+//       case 1:
+//         return "st";
+//       case 2:
+//         return "nd";
+//       case 3:
+//         return "rd";
+//       default:
+//         return "th";
+//     };
+//   };
+  
+//   outFirstTask.innerHTML = `${century}${getEnding(century)}`;
+// };
+
+// Task #2
+
+// document.querySelector('#second-btn').onclick = toBinary;
+// const outSecondTask = document.querySelector('.second-task');
+    
+// function toBinary() {

```

Comment:
It should have a name from the task
```suggestion
// function vowelOne() {

```